code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -279,3 +279,27 @@
 		slot_equipment_priority.Insert(index, slot_head)
 		return
 	return ..()
+
+/obj/item/weapon/reagent_containers/glass/beaker/waterbottle
+	name = "bottle of water"
+	desc = "A bottle of water filled at an old Earth bottling facility,"
+	icon = 'icons/obj/drinks.dmi'
+	icon_state = "smallbottle"
+	item_state = "bottle"
+	list_reagents = list("water" = 48, "fluorine" = 2)//see desc, don't think about it too hard

```

Comment:
So it contains toxin damaging reagents because of a political meme you're peddling? Literally putting poison in the vending machines for no reason